Phase4/ui.py

```diff
@@ -401,6 +401,85 @@ def avg_rating_branch_and_restaurant():  # Finds average rating for a Branch
     return
 
 
+def delete_dish():  # Delete a Dish
+    try:
+        dish = input("Enter Dish to be Deleted: ").strip()
+
+        cur.execute("DELETE FROM Dish_meal WHERE Dish_name = %s", (dish))
+        cur.execute("DELETE FROM Dish WHERE Dish_name = %s", (dish))
+        rows = cur.fetchall()
+        for row in rows:
+            print(row)
+        con.commit()
+        print()
+
+    except Exception as e:
+        con.rollback()
+        print(e)
+        print("-" * 10)
+
+    return
+
+
+def max_min_employee_rating_given_branch():  # Get Maximum or Minimum Rated Employee for a Branch
+    try:
+        choice = input(
+            "Enter 'Min' or 'Max' for Minimum or Maximum rating respectively: "
+        ).strip()
+        branch = input("Enter Branch: ").strip()
+
+        if choice == "Max":
```

Comment:
Make a dictionary or a conditional boolean flag and using xor remove redundant code.